css/properties/display.json

```diff
@@ -39,10 +36,13 @@
               "version_added": "1"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "1"
             },
             "samsunginternet_android": {
               "version_added": true
+            },
+            "webview_android": {
+              "version_added": "1"
```

Comment:
is there a version 1 for webview_android? There's no browser JSON for it so I'm not sure what the versioning system is like.

Reply:
needinfo?(@jpmedley)

Reply:
For something that old, I'd just put `true`. I don't have good information on versions that far back. 

Webview follows Chrome numbering.

Reply:
Well, Chrome for Desktop says `1` for that.